### PR TITLE
nginx: Update command to generate host config file

### PR DESCRIPTION
**Pull request by JohnCMoon**
Currently, instructions to copy the default configuration from the nginx container to the host uses three commands. This can be done more efficiently with one command (assuming the presence of "cat" in the container which should be a safe).

Update the instructions to use the single command.

### CODE REVIEW

**Comment by tianon** [comment]
cc `nginx` image maintainer @thresheek (and this would need to be moved over to `content.md` as `README.md` is generated)

**Comment by thresheek** [comment]
Thanks @JohnCMoon and @tianon !

I think the change looks good, so please fix the issues mentioned and we'll be good to go.

**Comment by JohnCMoon** [comment]
Moved change to `content.md` - sorry I missed the "do not edit" header at the top of `README.md`!

File: nginx/content.md [hunk 1]
```diff
@@ -42,9 +42,7 @@ You can mount your configuration file, or build a new image with it.
 If you wish to adapt the default configuration, use something like the following to get it from a running nginx container:
 
 ```console
-$ docker run --name tmp-nginx-container -d %%IMAGE%%
-$ docker cp tmp-nginx-container:/etc/nginx/nginx.conf /host/path/nginx.conf
-$ docker rm -f tmp-nginx-container
+$ docker run --rm --entrypoint=cat nginx /etc/nginx/nginx.conf > /host/path/nginx.conf
```
Comment (tianon) [inline nginx/content.md:45]: Doh sorry, didn't notice sooner -- this should be `%%IMAGE%%` so the templating uses the correct version on the per-arch pages too:

```suggestion
$ docker run --rm --entrypoint=cat %%IMAGE%% /etc/nginx/nginx.conf > /host/path/nginx.conf
```

Comment (tianon) [reply nginx/content.md:45]: (I'm happy to make this change for you if you'd prefer)

Comment (JohnCMoon) [reply nginx/content.md]: Good catch!

**Comment by tianon** [comment]
Thank you!

**Pull request merged**